content/posts/visualising-usage-using-batteries/index.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Battery Charts - Visualise usage rates & more"
+date: 2021-08-19T16:52:58+05:30
+draft: false
+description: A tutorial on how to show usage rates and more using batteries
+categories: ["tutorials"]
+displayInList: true
+author: Rithwik Rajendran
+resources:
+-   name: featuredImage
+    src: "Liverpool_Usage_Chart.png"
+    params:
+        description: "my image description"
+        showOnTop: true
+
+---
+
+# Introduction
+
+I have been creating the cliche visualisations like scatter plots, bar charts, beeswarms etc. for a while and thought about doing something different. Since I'm an avid football fan, I thought of ideas to represent players' usage or involvement over a period (a season, a couple of seasons). I have seen some cool visualisations like donut which depict usage and I wanted to make something different and simple to understand. I thought about representing batteries as a form of player usage and it made a lot of sense.
```

Comment:
I like that you're walking through your thought process here, but think the word cliche can be off putting - maybe common instead?

Reply:
Yes, "common" would be the better word here.

Reply:
Can you update cliche -> common? Then I think we're ready to publish it!